services/core-api/app/api/incidents/models/mine_incident_category.py

```diff
@@ -9,10 +9,11 @@
 
 class MineIncidentCategory(AuditMixin, Base):
     __tablename__ = 'mine_incident_category'
-    mine_incident_category_code = db.Column(db.String, primary_key=True)
+    mine_incident_category_code = db.Column(db.String(3), primary_key=True)
     description = db.Column(db.String, nullable=False)
+    display_order = db.Column(db.Integer, nullable=False)
     active_ind = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
 
     @classmethod
     def active(cls):
-        return cls.query.filter_by(active_ind=True).all()
+        return cls.query.filter_by(active_ind=True).order_by(cls.display_order).all()
```

Comment:
Not sure if we want to just return them sorted (what we're doing here) or just use the display_order attribute on the client-side to handle the sorting.